<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EmbySync</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="CPlayStateCompare" name="playStateCompare">
        <attribute name="title">
         <string>Played State</string>
        </attribute>
       </widget>
       <widget class="CUserInfoCompare" name="userInfoCompare">
        <attribute name="title">
         <string>User Info</string>
        </attribute>
       </widget>
       <widget class="CMissingEpisodes" name="missingEpisodes">
        <attribute name="title">
         <string>Missing Episodes</string>
        </attribute>
       </widget>
       <widget class="CMissingTVDBid" name="missingTVDBid">
        <attribute name="title">
         <string>Missing TVDBid</string>
        </attribute>
       </widget>
       <widget class="CMissingMovies" name="missingMovies">
        <attribute name="title">
         <string>Missing Movies</string>
        </attribute>
       </widget>
       <widget class="CCreateCollections" name="createCollections">
        <attribute name="title">
         <string>Create Collections</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Log:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPlainTextEdit" name="log">
          <property name="tabChangesFocus">
           <bool>true</bool>
          </property>
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1157</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuLoadRecent">
     <property name="title">
      <string>Load Recent Settings</string>
     </property>
     <addaction name="actionNoRecentFiles"/>
    </widget>
    <addaction name="actionLoadSettings"/>
    <addaction name="menuLoadRecent"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionCheckForLatestVersion"/>
   </widget>
   <widget class="QMenu" name="menuReload">
    <property name="title">
     <string>Reload</string>
    </property>
    <addaction name="actionReloadServers"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuReload"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Main</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadSettings"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>Reload</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionReloadServers"/>
  </widget>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="EmbySync.qrc">
     <normaloff>:/resources/settings.png</normaloff>:/resources/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionLoadSettings">
   <property name="icon">
    <iconset resource="EmbySync.qrc">
     <normaloff>:/resources/open.png</normaloff>:/resources/open.png</iconset>
   </property>
   <property name="text">
    <string>Load Settings...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="EmbySync.qrc">
     <normaloff>:/resources/save.png</normaloff>:/resources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionNoRecentFiles">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>No Recent Settings</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="toolTip">
    <string>Save As..</string>
   </property>
  </action>
  <action name="actionCheckForLatestVersion">
   <property name="text">
    <string>Check for Latest Version...</string>
   </property>
   <property name="toolTip">
    <string>Check for Latest Version...</string>
   </property>
  </action>
  <action name="actionReloadServers">
   <property name="icon">
    <iconset resource="EmbySync.qrc">
     <normaloff>:/resources/reloadServers.png</normaloff>:/resources/reloadServers.png</iconset>
   </property>
   <property name="text">
    <string>Reload Servers</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPlayStateCompare</class>
   <extends>QWidget</extends>
   <header>PlayStateCompare.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CUserInfoCompare</class>
   <extends>QWidget</extends>
   <header>UserInfoCompare.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CMissingEpisodes</class>
   <extends>QWidget</extends>
   <header>MissingEpisodes.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CMissingMovies</class>
   <extends>QWidget</extends>
   <header>MissingMovies.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CCreateCollections</class>
   <extends>QWidget</extends>
   <header>CreateCollections.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CMissingTVDBid</class>
   <extends>QWidget</extends>
   <header>MissingTVDBid.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="EmbySync.qrc"/>
 </resources>
 <connections/>
</ui>
